x/concentrated-liquidity/swapstrategy/zero_for_one_test.go

```diff
@@ -195,12 +195,36 @@ func (suite *StrategyTestSuite) TestComputeSwapStepOutGivenIn_ZeroForOne() {
 			expectedAmountOneOut:            osmomath.SmallestDec().MulInt64(100),
 			expectedSpreadRewardChargeTotal: osmomath.ZeroDec(),
 		},
+		"8 one ULP off from reaching target and one token zero in away (from fuzz seed 1694529692)": {
+			sqrtPriceCurrent:      types.MinSqrtPriceBigDec,
+			sqrtPriceTarget:       types.MinSqrtPriceV2,
```

Comment:
To clarify, this doesn't necessarily need to happen at the minSqrtPrice target right? Just the diff between these two needs to be a smallestBigDec?

Reply:
Should be possible. However, the fuzzer was hitting this at min tick only. I don't have other repros